tests/integration/checkboxradio/checkboxradio_core.js

```diff
@@ -0,0 +1,150 @@
+/*
```

Comment:
OHHHHHH!!!!! your PR i took over was before this was deleted so it added it back when merging!